src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -31,8 +31,8 @@ export const CentralImportDuplicateSearch = ({ duplicateSearchFilters }: Central
 
   const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')})` : '';
 
-  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
-    url: `${SearchApiPath.Registrations}?${searchQuery}&results=${maxHits}`,
+  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<ImportCandidate>>({
+    url: `${SearchApiPath.ImportCandidates}?${searchQuery}&results=${maxHits}`,
     errorMessage: t('feedback.error.search'),
   });
```

Comment:
Vi har gått fra å bruke `useFetch` til å bruke `reactQuery`. Denne har en del innebygd håndtering, som retries og `isLoading`.

```suggestion
  const importCandidateQuery = useQuery({
    queryKey: ['importCandidates', searchQuery],
    queryFn: () => fetchImportCandidates,
    onError: () => dispatch(setNotification({ message: t('feedback.error.get_messages'), variant: 'error' })),
  });
  
  const searchResults = importCandidateQuery.data ?? []; 
```